ccd-definition/CaseField.json

```diff
@@ -63,11 +63,36 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "UNSPECIFIED_CLAIMS",
     "ID": "claimType",
-    "Label": "CONTENT TBC: Enter the type of claim",
-    "FieldType": "FixedList",
+    "Label": "What type of claim is this?",
+    "FieldType": "FixedRadioList",
     "FieldTypeParameter": "ClaimType",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "claimTypeOther",
+    "Label": "Enter Claim type",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "personalInjuryType",
+    "Label": "What type of personal injury is this?",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "PersonalInjuryType",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "personalInjuryTypeOther",
+    "Label": "Enter personal injury type",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
```

Comment:
are we waiting on the spike to see how this can be modelled generically?

Reply:
I think it was agreed to go with what we have now and maybe change it depending on spike outcome